prometheus/Dockerfile

```
@@ -0,0 +1,5 @@
+FROM ghcr.io/deephaven/grpc-api
+COPY requirements.txt /
+COPY app.app /
+COPY prometheus.py /
+run pip3 install -r /requirements.txt
```

```suggestion
COPY app.d/ /app.d
RUN pip3 install -r /requirements.txt
```
make and app.d directory and place the 3 files in it?

---

Yeah that's way cleaner than what I have